app/Models/TrainingSession.php

```diff
@@ -101,6 +103,14 @@ public function retrieveStudents(): array
                 $status = 'unknown';
             }
 
+            $position_short_titles = $person->person_position
+                ->filter(fn($pp) => isset($positionsToReport[$pp->position_id]))
+                ->map(fn($pp) => $positionsToReport[$pp->position_id]->short_title !== ''
```

Comment:
The php way to check for empty strings is to use empty($value). Also, short_title can be null and empty() will also covert that case.

Reply:
nice, thanks and done